### ISSUE TITLE
How does one "Engage the Azure REST API Stewardship Board"?

### ISSUE THREAD

**Issue by PaulDempsey-ms** [Microsoft/api-guidelines#345]
In several places, these docs instruct teams to "Engage the Azure REST API Stewardship Board", but it doesn't explain exactly how one does that. Is that on Microsoft Teams?, email?, opening an issue here? You can't engage unless there is some venue to do so.

- [Microsoft Azure REST API Guidelines: Final Thoughts](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#final-thoughts)
- [Getting Help - The Azure REST API Stewardship Board](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#getting-help-the-azure-rest-api-stewardship-board)

**Comment by mikekistler** [comment]
@PaulDempsey-ms Fair point. We'll update the doc, but any of the means you mention -- teams, email, GitHub issue -- are all welcome.

**Comment by PaulDempsey-ms** [comment]
Great. Make sure to link to the Team, and provide the specific email address.
